### PR TITLE
Create serializer class for department and a view

### PR DESCRIPTION
**Pull request by habibaudu**
#### How should this be manually tested?
- Fire up postman
- Url: post `http://127.0.0.1:8000/api/v1/departments`
          `{"name":"Enter name","description":"enter department"}`
- Url : get `http://127.0.0.1:8000/api/v1/departments`
          - This gets all department
- Url: get `http://127.0.0.1:8000/api/v1/department/id/`
          - This gets a single department
- Url: patch `http://127.0.0.1:8000/api/v1/department/id/`
          - This updates a department
- Url : delete `http://127.0.0.1:8000/api/v1/department/id/`
          - This deletes a department
- Click `send` returns `OK`

### CODE REVIEW

File: ECommerceAPI/urls.py [hunk 1]
```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
```
Comment (el-Joft) [inline ECommerceAPI/urls.py:20]: why not set up a pylint that could help you to be consistent in your code e.g. single and double quote

File: ECommerceAPI/apps/department/department_url.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from django.urls import path, include
+from ECommerceAPI.apps.department import views
+
+urlpatterns = [
+    path("department/",views.DepartmentView.as_view()),
+    path("departments/<int:id>/",views.DepartmentDetailView.as_view())
```
Comment (el-Joft) [inline ECommerceAPI/apps/department/department_url.py]: `departments` as plural or `department` as singular.

File: ECommerceAPI/apps/department/serializer.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import re
```
Comment (el-Joft) [inline ECommerceAPI/apps/department/serializer.py:1]: what is this `re` doing?

Comment (habibaudu) [reply ECommerceAPI/apps/department/serializer.py:1]: Taking it out

File: ECommerceAPI/apps/department/department_url.py (hunk 2 above)
Comment (habibaudu) [reply ECommerceAPI/apps/department/department_url.py]: making the correction now

File: ECommerceAPI/urls.py (hunk 1 above)
Comment (habibaudu) [reply ECommerceAPI/urls.py:20]: will do

**Pull request merged**